Dockerfile

```diff
@@ -1,10 +1,10 @@
-# OpenJDK image with Java 1.8 JDK installed on an Alpine Linux base image.
+# AdoptOpenJDK image with Java 11 JDK installed on an Alpine Linux base image.
 # We want an Alpine Linux base image to keep the overall image size small.
-# We want a Java 1.8 JDK image as the verifier CLI needs a JDK (as opposed to a JRE).
+# We want a Java 11 JDK image as the verifier CLI needs a JDK (as opposed to a JRE).
 #
-# Ref: https://hub.docker.com/_/openjdk
-# Ref: https://hub.docker.com/layers/openjdk/library/openjdk/8-jdk-alpine/images/sha256-210ecd2595991799526a62a7099718b149e3bbefdb49764cc2a450048e0dd4c0?context=explore
-FROM openjdk:8-jdk-alpine
+# Ref: https://hub.docker.com/_/adoptopenjdk
+# Ref: https://hub.docker.com/layers/adoptopenjdk/openjdk11/alpine-slim/images/sha256-ef65f9b755ba9d70580d3b5e4ea7f133c68cecc096171959d011b38c4728f6b2?context=explore
+FROM adoptopenjdk/openjdk11:alpine-slim
```

Comment:
Hi @sadv1r - thank you for opening this issue and creating the PR!

Quick question, what's the motivation for changing the base image from `openjdk` to `adoptopenjdk/openjdk11`?

Reply:
There is no openjdk 11 alpine build

Reply:
Makes sense, thank you!